quickstarts/microsoft.logic/logic-app-create/metadata.json

```
@@ -13,6 +13,11 @@
     "resource": "Logic Apps, MySQL",
     "os": "Linux, Logic App",
     "level": "beginner",
-    "cost": "$"
+    "cost": "$",
+    "scenario": "Azure Logic App",
```

Fix the product name:

```suggestion
    "scenario": "Azure Logic Apps",
```